listView.php

```diff
@@ -355,10 +374,11 @@
 
 <script>
 
-// function to refresh the modal page for task details
+// function to refresh the modal page for task details and filtering task
 function dynamicModal(str)
 {
 $("#viewTaskFrame").attr("src", "https://mansci-db.uwaterloo.ca/~wmmeyer/viewTask.php?id="+str);
```

Comment:
Will be merge conflict with this since it is removed in another pull request.

Reply:
It's not showing any conflicts as of now - has the PR been made already, or is it just a commit?

Reply:
At the moment it is just a commit in another branch. The commit is a change to the structure of viewing a task which no longer requires this function. It can be kept for now and removed in the merge conflict later.